climada/engine/forecast.py

```diff
@@ -144,15 +153,16 @@ def __init__(
             as long as the attribute Hazard.date is the same for all
             events. Several run_datetime:Hazard combinations for the same
             event can be provided.
-        exposure : Exposure
+        exposure : Exposures
         impact_funcs : ImpactFuncSet
         haz_model : str, optional
             Short string specifying the model used to create the hazard,
             if possible three big letters. Default is 'NWP' for numerical
             weather prediction.
         exposure_name : str, optional
             string specifying the exposure (e.g. 'EU'), which is used to
-            name output files.
+            name output files. If ``None``, the name will be inferred from the region ID
+            in the Exposures GeoDataframe. If that fails, it defaults to ``"custom"``.
```

Comment:
```suggestion
            name output files. If ``None``, the name will be the name of the country
            corresponding to the smallest region ID (ISO3-numeric) value in the Exposures
            GeoDataframe. If that fails, it defaults to ``"custom"``.
```

Reply:
It is clearer, thanks! Yet it misses the reference to the `region_id` column. There are no ISO code in an exposures geodataframe by default. However, the region_id column is by default used to identify countries using their ISO code.

so `in the Exposures' GeoDataframe column region_id` maybe?

Reply:
```suggestion
            name output files. If ``None``, the name will be inferred from the Exposures
            GeoDataframe ``region_id`` column, using the corresponding name of the region
            with the lowest ISO 3166-1 numeric code. If that fails, it defaults to
            ``"custom"``.
```